teachers_digital_platform/css/organisms/filter-panel.less

```diff
@@ -0,0 +1,48 @@
+// Filter panel
+
+.o-filter-panel {
+    .tdp-activity-search & {
+        background-color: #f7f8f9;
+        border: 1px solid #ccc;
+        padding: 30px;
```

Comment:
Please replace with an em value. We like using a calculation to make it easy for future viewers of the code to see what the corresponding pixel value is intended to be, e.g.,

```less
padding: unit( 30px / @base-font-size-px, em );
```

Reply:
Switched to 2em.